PabloPS0/StockManager
# chore: update database connection, remove old files, and add .env example

### Context
This PR addresses changes and improvements related to the database configuration and repository structure. The main changes include the creation of a new .env.example file, the update of the database connection path, the removal of old database files, and the addition of the database file to .gitignore to prevent it from being versioned.

### Changes Made
- Created .env.example file: Added an example file containing the environment variable to configure the database connection path. This file serves as a template for developers.
- Updated database connection path: Modified the database connection path to reflect the new project structure.
- Removed old database file: The old database file has been deleted to avoid confusion and keep the repository clean.
- Added database file to .gitignore: The database file has been added to .gitignore to ensure it is not tracked or versioned in the repository.
### Testing
- Copy the .env.example file to a new .env file in your local environment and configure the correct database connection path.
- Check if the connection path was updated correctly and the application can connect to the database.
- Test the application to ensure the old database file was removed and the connection to the new database is working correctly.
### Notes
- There are no functional changes to the code other than updates related to the database configuration.
- The removal of the old database file and its addition to .gitignore helps to keep the repository clean and organized.
- Make sure to configure the .env file correctly in your local environment.

### Reference
This PR contains changes based on the `feature/test-experimental` branch.
